README.md

```diff
@@ -10,6 +10,31 @@ The majority of documentation on the site is drawn from [the probot docs](https:
 
 We also use this website to show off some of the cool probot plugins built by the community. If you would like to have your plugin listed there, check out our [Contributing Guidelines](https://github.com/probot/probot.github.io/blob/master/CONTRIBUTING.md) for more information about what the requirements are to do that.
 
+## Install and Usage
+
+If you'd like to work on this repository, clone it. We use [Jekyll](https://jekyllrb.com/) to build the website, hosted on [GitHub Pages](https://pages.github.com/). We also use Git submodules to pull docs from the docs folder in  [probot/probot](https://github.com/probot/probot/tree/master/docs).
+
+Once you've cloned this repo, run these commands in this directory:
+
+```sh
+# Only needed the first time:
+script/bootstrap
+# And each time you work locally:
+script/server
+```
+
+You should then be able to navigate to the URL printed in your console, and see live edits you make render in your browser. You'll likely have to familiarize yourself with how Jekyll works to do larger edits, but it's worth the effort!
+
+Another note: You can see more commands we use, such as our native build and server functions, in `script`. Go take a look.
+
+## Contribute
+
+We'd love to have you contribute! PRs are gladly accepted, and issues are also great. If you're thinking about a major PR, it would be better to open an issue first to talk about it.
+
+If you're interested in contributing, check out our [contributing docs](CONTRIBUTING.md) to get started.
+
+Want to get more involved with the Probot community? [Take a look at our community page](https://probot.github.io/community/)!
+
 ## License
 
-This website is licensed [CC-BY-4.0](LICENSE). The Probot logo is licensed [CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.en). It was found on [wikimedia.org](https://commons.wikimedia.org/wiki/File:Robot-clip-art-book-covers-feJCV3-clipart.png) and is from clipartkid.com.
+This website is licensed [CC-BY-4.0](LICENSE) © 2018 Brandon Keepers. The Probot logo is licensed [CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.en). It was found on [wikimedia.org](https://commons.wikimedia.org/wiki/File:Robot-clip-art-book-covers-feJCV3-clipart.png) and is from clipartkid.com.
```

Comment:
@bkeepers Does the license need to change, yet? I know there was some discussion about this now that Probot is becoming more of an official GitHub project.

Reply:
Well, at the moment it isn't attributed to anyone. GitHub would also work for the licensee.

Reply:
> Does the license need to change, yet? I know there was some discussion about this now that Probot is becoming more of an official GitHub project.

🔜